PowerEditor/installer/nativeLang/polish.xml

```diff
@@ -872,41 +969,124 @@
+Ustawienia chmury zostaną wycofane. Nową ścieżkę można podać w oknie preferencji."/>
+            <FilePathNotFoundWarning title="Otwarcie pliku" message="Wybrany plik nie istnieje."/>
+            <SessionFileInvalidError title="Ładowanie sesji" message="Nie udało się załadować sesji. Plik sesji jest uszkodzony lub nieprawidłowy."/>
+            <DroppingFolderAsProjectModeWarning title="Nieprawidłowe działanie" message="Pliki i foldery muszą być przenoszone do programu osobno, ponieważ aktywny jest tryb „Folder jako Projekt”.
+Żeby to zmienić, należy włączyć opcję „Nie traktuj przenoszonego folderu jako obszaru roboczego, tylko ładuj wszystkie jego pliki” w zakładce „Domyślna ścieżka” w oknie preferencji."/>
+            <SortingError title="Błąd sortowania" message="Nie można wykonać sortowania przez linię numer $INT_REPLACE$."/>
+            <ColumnModeTip title="Wskazówka dotycząca edycji kolumnowej" message="ALT+LPM oraz Alt+Shift+Strzałka przełączają tryb kolumnowy."/>
+            <BufferInvalidWarning title="Błąd zapisywania" message="Nie udało się zapisać. Błąd bufora."/>
+            <DoSaveOrNot title="Zapisywanie" message="Czy zapisać plik „$STR_REPLACE$”?"/>
+            <DoCloseOrNot title="Zachowywanie nieistniejącego pliku" message="Plik „$STR_REPLACE$” nie istnieje.
+Czy mimo to zachować go w edytorze?"/>
+            <DoDeleteOrNot title="Usuwanie" message="Plik „$STR_REPLACE$”
+zostanie zamknięty i przeniesiony do Kosza.
 Kontynuować?"/>
-            <SettingsOnCloudError title="Błąd ustawień w chmurze" message="Ścieżka do ustawień w chmurze jest tylko do odczytu
-lub wymagane są prawa do zapisu.
-Twoje ustawienia w chmurze zostaną anulowane. Zmień miejsce zapisu ustawień przez okno preferencji."/>
+            <NoBackupDoSaveFile title="Zapisywanie" message="Nie udało się odnaleźć kopii zapasowej na dysku. Została usunięta.
+Żeby nie stracić danych, należy zapisać bieżące zmiany.
+Zapisać plik „$STR_REPLACE$”?"/>
+            <DoReloadOrNot title="Przeładowywanie" message="Plik „&quot;$STR_REPLACE$”
+
+został zmodyfikowany przez inny program.
+Przeładować go?"/>
+            <DoReloadOrNotAndLooseChange title="Przeładowywanie" message="Plik „$STR_REPLACE$”
+
+został zmodyfikowany przez inny program.
+Przeładować go i stracić wszystkie zmiany dokonane w Notepad++?"/>
+            <PrehistoricSystemDetected title="Antyczny system" message="Wykryto antyczny system operacyjny. Przepraszamy, ale ta funkcja działa jedynie na nowoczesnych systemach."/>
+            <XpUpdaterProblem title="Samoaktualizacja Notepad++" message="Automatyczne aktualizowanie Notepad++ jest niemożliwe w systemie Windows XP ze względu na przestarzałe sposoby ochrony systemu.
+Uruchomić stronę internetową Notepad++ z najnowszą wersją programu?"/>
+            <DocTooDirtyToMonitor title="Problem z monitorowaniem" message="Plik został naruszony. Musi zostać pierw zapisany, żeby mógł być monitorowany."/>
```

Comment:
"pierw" is archaic. "wpierw" would be a good compromise.

Reply:
Highly debatable. Different linguists categorise "wpierw" differently. I, myself, see it as archaic or more literary and see nothing wrong with "pierw", while for example my Słownik poprawnej polszczyzny (a book about proper use of Polish) defines "wpierw" as a colloquial version of "najpierw" ("firstly, beforehand"). I'm leaving it as is, because it's not wrong per se.